main.asm

```diff
@@ -1,166 +1,185 @@
 INCLUDE "hardware.inc"
 INCLUDE "ibmpc1.inc"
 ; rst vectors
-SECTION "Dummy RST Handlers",ROM0[0]
-REPT 8
-	ret
-REPT 7
-	nop
-ENDR
-ENDR
+SECTION "RST Handlers",ROM0[$0000]
+fillmem equ $00
+_fillmem:
+    inc b
+    inc c
+    jr .skip
+.loop
+    ld [hli], a
+.skip
+    dec c
+    jr nz, .loop
+    dec b
+    jr nz, .loop
+    ret
+    ds 4
 
-SECTION "vblank",ROM0[$40]
-	jp VBlank
-SECTION "lcdc",ROM0[$48]
-	reti
-SECTION "timer",ROM0[$50]
-	reti
-SECTION "serial",ROM0[$58]
-	reti
-SECTION "joypad",ROM0[$60]
-	reti
+memcpy equ $10
+_memcpy:
+    inc b
+    inc c
+    jr .skip
+.loop
+    ld a, [hli]
+    ld [de], a
+    inc de
+.skip
+    dec c
+    jr nz, .loop
+    dec b
+    jr nz, .loop
+    ret
+    ds 2
 
-SECTION "signature",ROM0[$80]
-	db "MineRobber9000"
+REPT 4
+    ret
+    ds 7
+ENDR
 
-SECTION "romheader",ROM0[$100]
-	nop
-	jp Start
+VBlank: ; VBlank handler here
+    call Joypad
+    call DisplayJoypadState
+    ldh a, [hSendByLinkCable]
+    rra ; Bit 0 into carry
+    ldh a, [hCurrentJoypadState]
```

Comment:
This is the link cable sending fix.

Reply:
What exactly does this fix? It seems to just do what is already being done, but more optimized.

Reply:
NVM, saw your post in #tools.